test/elvis.config

```diff
@@ -5,46 +5,21 @@
     {config,
      [#{dirs => ["../../src", "../../test"],
         filter => "*.erl",
-        rules => [{elvis_style, line_length, #{limit => 80,
-                                               skip_comments => false}},
-                  {elvis_style, no_tabs},
-                  {elvis_style, no_trailing_whitespace},
-                  {elvis_style, macro_names},
-                  {elvis_style, macro_module_names},
-                  {elvis_style, operator_spaces, #{rules => [{right, ","},
-                                                             {right, "++"},
-                                                             {left, "++"}]}},
-                  {elvis_style, nesting_level, #{level => 3}},
-                  {elvis_style, god_modules, #{limit => 25}},
-                  {elvis_style, no_if_expression},
+        ruleset => erl_files,
+        rules => [
                   {elvis_style, invalid_dynamic_call, #{ignore => [ktn_recipe_verify]}},
-                  {elvis_style, used_ignored_variable},
-                  {elvis_style, no_behavior_info},
-                  {
-                    elvis_style,
-                    module_naming_convention,
-                    #{regex => "^([a-z][a-z0-9]*_?)*(_SUITE)?$",
-                      ignore => []}
-                  },
-                  {
-                    elvis_style,
-                    function_naming_convention,
-                    #{regex => "^([a-z$][a-z0-9]*_?)*$"}
-                  },
-                  {elvis_style, state_record_and_type},
-                  {elvis_style, no_spec_with_records},
                   {elvis_style, dont_repeat_yourself, #{min_complexity => 15}},
                   {elvis_style, no_debug_call, #{debug_functions => [{ct, pal}]}}
                  ]
        },
       #{dirs => ["../.."],
         filter => "Makefile",
-        rules => [{elvis_project, no_deps_master_erlang_mk, #{ignore => []}},
-                  {elvis_project, protocol_for_deps_erlang_mk, #{ regex => "(https://.*|[0-9]+([.][0-9]+)*)"}}]
+        ruleset => makefiles,
+        rules => [{elvis_project, protocol_for_deps_erlang_mk, #{ regex => "(https://.*|[0-9]+([.][0-9]+)*)"}}]
```

Comment:
Do we even have a Makefile still? We should consider just removing that thing. Support for `erlang.mk` is not needed anymore.

Reply:
9feef7a